backend/src/main/scala/com/softwaremill/adopttapir/starter/StarterModel.scala

```diff
@@ -5,7 +5,8 @@ case class StarterDetails(
     groupId: String,
     serverEffect: ServerEffect,
     serverImplementation: ServerImplementation,
-    tapirVersion: String
+    tapirVersion: String,
+    documentationAdded: Boolean
```

Comment:
[minor, but naming] -> `addDocumentation`? it's not yet added ;)

Reply:
makes sense